docs/integrators/deep-history-squad.md

```diff
@@ -102,10 +102,13 @@ Since each observer of a deep-history squad must have a non-pruned history, thei
 
 [comment]: # (mx-context-auto)
 
-## Observer configuration
-
-A deep history squad has its observers set up to retain the whole, non-pruned history of the blockchain. That is, the observers must be started with the flag `--operation-mode=historical-balances`.
+## Observer installation and configuration
 
+The installation of a deep history squad is the same as that of a [regular observing squad](/integrators/observing-squad).
```

Comment:
> is the same

> the difference is that

Maybe `is almost the same` instead of `the same`?

Reply:
Done.